go-chaos/worker/chaos_worker_test.go

```diff
@@ -116,11 +116,31 @@ func Test_ShouldSendExperimentsForClusterPlan(t *testing.T) {
 	assert.Equal(t, experiments, fakeJobClient.Variables)
 }
 
+func Test_ShouldFailJobForNonExistingClusterPlan(t *testing.T) {
```

Comment:
:x: This is testing the same thing as the following test `Test_ShouldFailWhenNoClusterPlanForReadExperimentsJob`

Reply:
Ups I have overseen this somehow. Thanks :+1: